commit f498b46e7624875d47ac5c94683a6c04bc2f6001 (refs/changes/00/600/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-05T12:34:57-07:00 (3 years ago)
    
    WORKFLOW-212 get wf-client working with node v4, and publish versions to npm (ignore file so cutarelease can work)

diff --git a/.gitignore b/.gitignore
index ce891c9..48bc125 100644
--- a/.gitignore
+++ b/.gitignore
@@ -7,3 +7,4 @@ cscope.in.out
 cscope.po.out
 cscope.out
 smf/manifests/bapi.xml
+/npm-debug.log
